common-lib/src/main/scala/com/gu/mediaservice/lib/play/RequestLoggingFilter.scala

```diff
@@ -5,27 +5,39 @@ import akka.stream.Materializer
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success}
 import play.api.mvc.{Filter, RequestHeader, Result}
-import play.api.Logger
-import com.gu.mediaservice.lib.metrics.StopWatch
+import play.api.{Logger, MarkerContext}
+import net.logstash.logback.marker.Markers.appendEntries
+
+import scala.collection.JavaConverters._
 
 class RequestLoggingFilter(override val mat: Materializer)(implicit ec: ExecutionContext) extends Filter {
 
   private val logger = Logger("request")
 
   override def apply(next: (RequestHeader) => Future[Result])(rh: RequestHeader): Future[Result] = {
-    val stopWatch = new StopWatch
-
+    val start = System.currentTimeMillis()
     val result = next(rh)
 
     val originIp = rh.headers.get("X-Forwarded-For").getOrElse(rh.remoteAddress)
     val referer = rh.headers.get("Referer").getOrElse("")
     result onComplete {
       case Success(response) =>
         val length = response.header.headers.getOrElse("Content-Length", 0)
-        logger.info(s"""$originIp - "${rh.method} ${rh.uri} ${rh.version}" ${response.header.status} $length "$referer" ${stopWatch.elapsed}ms""")
+
+        val elapsed = System.currentTimeMillis() - start
+        val markers = MarkerContext(appendEntries(Map(
+          "duration" -> elapsed
```

Comment:
You want to take this opportunity to pull anything else out here e.g. status code?

Reply:
I've added more fields